app/assets/javascripts/discourse/app/static/wizard/components/fields/styling-preview/-homepage-preview.js

```diff
@@ -44,7 +44,7 @@ export default class HomepagePreview extends PreviewBaseComponent {
         homepageStyle
       )
     ) {
-      this.drawPills(colors, font, height * 0.15, { homepageStyle });
+      this.drawPills(colors, font.label, height * 0.15, { homepageStyle });
```

Comment:
It looks like `drawPills()` expects the font object, not the font label?

Reply:
Ah yes sorry, typo